src/Maatwebsite/Excel/Writers/LaravelExcelWriter.php

```diff
@@ -350,6 +365,8 @@ protected function _download(Array $headers = [])
      */
     public function store($ext = 'xls', $path = false, $returnInfo = false)
     {
+        $this->checkExtensionIsValid($ext);
```

Comment:
`download()` calls `export()`. I added method call in `export() and in `store()`.